classes/payment_manager.php

```diff
@@ -60,39 +62,42 @@ public static function paymentplugin_enrol(string $courseid, string $userid) {
 
     /**
      * Actions a transaction given the correct data.
-     * 
+     *
      * @param int $paymentstatus Either PAYMENT_FAILED, PAYMENT_COMPLETE or PAYMENT_INCOMPLETE
-     * @param string $gateway_table_name The name of the subplugin table where necessary non default data will be sent too. If null, no additional data will be saved to the database.
+     * @param string $gatewaytablename The name of the subplugin table where necessary non default data will be sent too.
```

Comment:
The gateway table name (and gateway name) is stored in the paymentgateway object now, so just be careful of that when merging